### PR TITLE
USAGOV-2150-es-wizard-crumb: Removes the if-clause to prepend Spanish…

### PR DESCRIPTION
**Pull request by omerida**
… path by asking Drupal for the Breadcrumb URI in the expected language.

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-2150

## Description

Removes the need to check and prepend `/es/` to Spanish wizard breadcrumbs

## Type of Changes

- [ ] New Feature
- [x] Bugfix - Refactor
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [ ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] WWW
  - [ ] Egress
  - [ ] Tools
  - [ ] Cron
- [ ] Other

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [ ] Requires New Config
- [ ] Requires New Content

### Validation Steps

Same verification steps as 2147

1. Run bin/drush cr to ensure all caches are cleared
2. Visit these links and check the breadcrumbs. All should have [Página principal ](http://localhost/es/)> [Estafas y fraudes ](http://localhost/es/estafas-y-fraudes)> Aprenda dónde reportar o denunciar una estafa (specifically, the last breadcrumb should not be a link to the Spanish home page, as it currently is in our dev and stage environments):

-     http://localhost/es/donde-reportar-una-estafa
-     http://localhost/es/donde-reportar-una-estafa/donde-ocurrio-estafa
-     http://localhost/es/donde-reportar-una-estafa/categoria-tipo-estafa

3. Extra credit: check the result of running tome on this branch against dev.

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: web/modules/custom/usagov_wizard/usagov_wizard.module [hunk 1]
```diff
@@ -408,22 +408,22 @@ function usagov_wizard_get_term_breadcrumb(Term $rootTerm): array {
   if (isset($rootTerm->field_wizard_breadcrumbs)) {
     $crumbs = $rootTerm->field_wizard_breadcrumbs->getValue();
     $result = [];
+
     foreach ($crumbs as $crumb) {
       if (isset($crumb['uri'])) {
-        $crumb['url'] = Url::fromUri($crumb['uri'])->toString();
-        // Sometimes Drupal doesn't prepend Spanish path prefix.
         // Sometimes this crumb's uri is "route:<nolink>" and the url is an empty string;
```
Comment (akf) [inline web/modules/custom/usagov_wizard/usagov_wizard.module]: Please remove this comment too! 
(I'm just getting into the tome-running and ppr-reviewing, so you'll probably get to that before I'm ready to approve.)

Comment (omerida) [reply web/modules/custom/usagov_wizard/usagov_wizard.module]: Remove this comment?

```
// Sometimes this crumb's uri is "route:<nolink>" and the url is an empty string;
// in that case we should leave it empty (current page, not to be linked).
```

Comment (akf) [reply web/modules/custom/usagov_wizard/usagov_wizard.module]: Yes

Comment (omerida) [reply web/modules/custom/usagov_wizard/usagov_wizard.module]: removed

**Review by akf — CHANGES_REQUESTED** [review]
Pages look good, Tome output is good. 
The published-pages.csv has a problem: the /estafas-y-fraudes "taxonomy level 2" entry for wizard terms lacks the "/es". I see this code path being exercised for that breadcrumb, and that it actually doesn't generate a path with /es even though the language is provided.

**Review by akf — APPROVED** [review]
Produces no change in HTML output nor in the PPR (as desired).

**Pull request merged**